io.catenax.user_estimated_loading/1.0.0/UserEstimatedLoading.ttl

```diff
@@ -0,0 +1,244 @@
+#######################################################################
+# Copyright (c) 2022 Allgemeine Deutsche Automobil-Club (ADAC) e.V
+# Copyright (c) 2022 ZF Friedrichshafen AG
+# Copyright (c) 2022 Contributors to the Eclipse Foundation
+#
+# See the NOTICE file(s) distributed with this work for additional
+# information regarding copyright ownership.
+#
+# This work is made available under the terms of the 
+# Creative Commons Attribution 4.0 International (CC-BY-4.0) license, 
+# which is available at
+# https://creativecommons.org/licenses/by/4.0/legalcode.
+#
+# SPDX-License-Identifier: CC-BY-4.0
+#######################################################################
+
+@prefix samm: <urn:samm:org.eclipse.esmf.samm:meta-model:2.0.0#> .
+@prefix samm-c: <urn:samm:org.eclipse.esmf.samm:characteristic:2.0.0#> .
+@prefix samm-e: <urn:samm:org.eclipse.esmf.samm:entity:2.0.0#> .
+@prefix unit: <urn:samm:org.eclipse.esmf.samm:unit:2.0.0#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix : <urn:samm:io.openmanufacturing.digitaltwin:1.0.0#> .
+
+:UserEstimatedLoading a samm:Aspect ;
+   samm:preferredName "User defined loading"@en ;
+   samm:description "user defined list of usage with percentage, driving style, eniviroment (hilliness, city_use, trailer_use)"@en ;
+   samm:properties ( :vehicleStatus :vehicleData :history :future ) ;
+   samm:operations ( ) ;
+   samm:events ( ) .
+
+:vehicleStatus a samm:Property ;
+   samm:preferredName "current status of vehicle"@en ;

```

Comment:
would write preferredName with uppercase letters as it is used a s proper noun, ie Current Status of Vehicle or alternatively as you did ini the Aspect definition start with an uppercase letter. Whichever approach you use, be consistent in all prefered names throughout the model